hazelcast/src/main/java/com/hazelcast/client/impl/clientside/HazelcastClientInstanceImpl.java

```diff
@@ -869,13 +869,13 @@ public JetService getJet() {
         return clientExtension.getJet();
     }
 
-    public void onClusterChange() {
+    public void onNextCluster() {
```

Comment:
Minor: I believe the old method name was a bit better

Reply:
I think onClusterChange does not imply: "we change the cluster we are currently trying to connect to." I thought of some names but nothing short and simple come to my mind. We can name it `onTryToConnectNextCluster` WDYT?

Reply:
I think onNextCluster clashes with my new method `onConnectionToNewCluster`

Reply:
`onTryToConnectNextCluster` looks better to me, compared to `onNextCluster`

Reply:
Changed it to onTryToConnectNextCluster